metrics.py

```diff
@@ -0,0 +1,44 @@
+import enum
+
+import prometheus_client
+
+class Metrics(enum.Enum):
```

Comment:
lets add a metric for
1. 511 api response codes
2. 511 api latency (to track how long the apis took to respond
3. cache last updated (we store a timestamp to this metric every time it updates)
4. http response code (increment with label each time we return 200/400 etc

Reply:
done